packages/react-reconciler/src/ReactProfilerTimer.js

```diff
@@ -158,10 +158,10 @@ export function startUpdateTimerByLane(
       const newEventTime = resolveEventTimeStamp();
       const newEventType = resolveEventType();
       if (
-        newEventTime !== blockingEventTime ||
+        newEventTime !== blockingEventRepeatTime ||
         newEventType !== blockingEventType
       ) {
-        blockingEventIsRepeat = false;
+       blockingEventRepeatTime = -1.1;
```

Comment:
```suggestion
        blockingEventRepeatTime = -1.1;
```

Hopefully this fixes the linter